assets/models/system/Mistral-7B-Instruct-v0-1/spec.yaml

```diff
@@ -0,0 +1,44 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Mistral-7B-Instruct-v0-1
+path: ./
+properties:
+  SHA: f966a600a1f2c35a067dcb16b24bbf9ac3b85799
+  inference-recommended-sku: Standard_E16s_v3, Standard_NC12s_v3, Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  evaluation-recommended-sku: Standard_NC12s_v3, Standard_NC24s_v3, Standard_NC24rs_v3,
+    Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  finetune-recommended-sku: Standard_ND40rs_v2, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4
+  finetuning-tasks: text-classification, text-generation
+  languages: EN
+  azureml.copyRegistryFilesToWorkspace: true
+tags:
+  Featured: ''
+  Preview: ''
+  evaluation_compute_allow_list:
+  - Standard_NC12s_v3
+  - Standard_NC24s_v3
+  - Standard_NC24rs_v3
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_A100_v4
+  inference_compute_allow_list:
```

Comment:
Let's more add SKUs here. Specifically, please include all the  A100 SKUs. 

Standard_NC48ads_A100_v4
Standard_NC96ads_A100_v4
Standard_ND96asr_v4
Standard_ND96amsr_A100_v4
Standard_NC24ads_A100_v4

Reply:
Thank you. Added.